demo/index.html

```diff
@@ -4,7 +4,7 @@
 		<meta charset="utf-8">
 		<title>Typography Snapshot</title>
 		<meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
-		<script src="../../webcomponentsjs/webcomponents-lite.min.js"></script>
+		<script src="../../webcomponentsjs/webcomponents-lite.js"></script>
```

Comment:
@dlockhart Note; I did not update to use the webcomponents-loader.js as i think this needs to be a reference that will work for both the 1.x and 2 variants